data/sources.config

```diff
@@ -120,6 +120,7 @@ plist:key	PLIST	Plist Entry
 PLSRecall:event	PLSRecall	PL/SQL Developer Recall file
 popularity_contest:log:event	LOG	Popularity Contest Log
 popularity_contest:session:event	LOG	Popularity Contest Session
+postgresql LOG PostgreSQL Log
```

Comment:
i believe this should be postgresql:line

Reply:
Please use `postgresql:log:entry` or `postgresql:log:line`

Reply:
since it is a multi line log `entry` is likely better so maybe `postgresql:application_log:entry`

Reply:
Done